apps/server/lib/lexical/server/iex/helpers.ex

```diff
@@ -181,4 +186,17 @@ defmodule Lexical.Server.IEx.Helpers do
   defp ensure_project(project) when is_atom(project) do
     project(project)
   end
+
+  defp trans(name, function) do
+    case :persistent_term.get(name, :undefined) do
```

Comment:
This name is really unlikely to ever conflict with something else in `:persistent_term`, but it's easy enough to fix here just to be sure.

```suggestion
  defp trans(name, function) do
    name = {__MODULE__, name}
    case :persistent_term.get(name, :undefined) do
```